packages/atomic/cypress/integration/search-box.cypress.ts

```diff
@@ -36,7 +36,8 @@ describe('Search Box Test Suites', () => {
     const numOfRecentQueries = 4;
     const maxSuggestionsWithoutQuery = numOfSuggestions - 1;
     const maxRecentQueriesWithoutQuery = numOfRecentQueries - 1;
-    beforeEach(() => {
+
+    function setupWithSuggestionsAndRecentQueries() {
       new TestFixture()
         .with(setSuggestions(numOfSuggestions))
```

Comment:
The order is important here, the way they appear in the suggestion should be the way they are defined in the DOM as slots

Reply:
I experimented with both orders by adjusting these [if-blocks](https://github.com/coveo/ui-kit/blob/master/packages/atomic/cypress/integration/search-box-actions.ts#L21), but in both cases the recent queries were cleared on the first `keydown` 😶